src/axom/primal/operators/clip.hpp

```diff
@@ -120,15 +120,18 @@ Polygon<T, 3> clip(const Triangle<T, 3>& tri, const BoundingBox<T, 3>& bbox)
  * \param [in] checkSign If true (default is false), checks if the
  *             signed volume of each shape is positive. If the signed volume
  *             of that shape is negative, order of some vertices will be
- *             swapped for that shape.
+ *             swapped to try to obtain a positive volume
+ *             for that shape. Otherwise, the returned polyhedron
```

Comment:
Please disambiguate the subject of "otherwise" in these doxy-comments. (See my previous comment).